app/src/controllers/organizations/projects/constants.js

```diff
@@ -21,6 +21,7 @@ export const NAMESPACE = 'projects';
 export const DEFAULT_LIMITATION = 300;
 export const DEFAULT_OFFSET = 0;
 export const DEFAULT_SORT_COLUMN = 'name';
+export const DEFAULT_PROJECTS_PAGE_SIZE = 20;
```

Comment:
That's the previous name of the `limit` for pagination.
Please refer the `src/controllers/instance/events` implementation for details on how the new pagination properties look like.

Reply:
Done.